src/Configuration/PlatesFormatterConfiguration.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace Equip\Configuration;
+
+use Auryn\Injector;
+use Equip\ContentNegotiation;
+use Equip\Formatter\PlatesFormatter;
+
+class PlatesFormatterConfiguration implements ConfigurationInterface
```

Comment:
Yeah, I think that was part of my line of thinking in [this comment](https://github.com/equip/framework/pull/76#issuecomment-237586730), I just wasn't sure how to articulate it at the time. Thanks for that @bvisness. 😄